kafka/manifest.json

```diff
@@ -1,54 +1,67 @@
 {
-  "categories": [
-    "processing",
-    "messaging",
-    "log collection",
-    "autodiscovery"
-  ],
-  "creates_events": false,
-  "display_name": "Kafka",
-  "guid": "f201c0b7-4b31-4528-9955-ae756a4580b8",
-  "is_public": true,
-  "maintainer": "help@datadoghq.com",
-  "manifest_version": "1.0.0",
-  "metric_prefix": "kafka.",
-  "metric_to_check": "kafka.net.bytes_out.rate",
-  "name": "kafka",
-  "process_signatures": [
-    "java kafka.kafka"
-  ],
-  "public_title": "Datadog-Kafka Integration",
-  "short_description": "Collect metrics for producers and consumers, replication, max lag, and more.",
-  "support": "core",
-  "supported_os": [
-    "linux",
-    "mac_os",
-    "windows"
-  ],
-  "type": "check",
-  "integration_id": "kafka",
+  "manifest_version": "2.0.0",
+  "app_uuid": "39640d5e-54be-48ff-abf1-8871499e2fd3",
+  "app_id": "kafka",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect metrics for producers/consumers, replication, lag, and more.",
```

Comment:
```suggestion
    "description": "Collect metrics for producers and consumers, replication, lag, and more.",
```

Reply:
There is now a 70 character limit so tiles look nicer 🙁

Reply:
nvm! https://github.com/DataDog/integrations-core/pull/12339